rocm_agent_enumerator

```diff
@@ -196,7 +196,7 @@ def readFromKFD():
       if os.path.isdir(node_path):
         prop_path = node_path + '/properties'
         if os.path.isfile(prop_path) and os.access(prop_path, os.R_OK):
-          target_search_term = re.compile("gfx_target_version.+")
+          target_search_term = re.compile(r"gfx_target_version.+")
```

Comment:
There is no practical difference between using a raw string and a non-raw string, as there are no backslash escape characters involved here.

Reply:
Thank you for reviewing. Yes, both normal string and raw string can be used here to achieve similar result.

What's your opinion? I think making all re.compile use raw strings prevents making mistakes in the future (if regular expressions is needed here). While reverting these 3 lines makes this commit minimal.

Reply:
Sure. I agree that using raw string will prevent future mistakes incase escape characters are added here. Please go ahead with the commit.